tools/add_triage_signature.py

```diff
@@ -201,13 +201,29 @@ def get_host_log_file(triage_logs_tar, host_id, filename):
     # The file is already uniquely determined by the host_id, we can omit the hostname
     hostname = "*"
 
-    return triage_logs_tar.get(f"{hostname}.tar.gz/logs_host_{host_id}/{filename}")
+    new_logs_path = f"{hostname}.tar/{hostname}.tar.gz/logs_host_{host_id}/{filename}"
+    old_logs_path = f"{hostname}.tar.gz/logs_host_{host_id}/{filename}"
+    try:
+        agent_logs = triage_logs_tar.get(new_logs_path)
+    except FileNotFoundError:  # backward compatibility
+        # TODO(MGMT-13705): Remove this when MGMT-13454 is in production
+        logger.warning(f"Could not find logs under {new_logs_path}, attempting {old_logs_path}")
+        return triage_logs_tar.get(old_logs_path)
+
+    return agent_logs
 
 
 def get_journal(triage_logs_tar, host_ip, journal_file):
-    return triage_logs_tar.get(
-        f"*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane/{host_ip}/journals/{journal_file}"
-    )
+    new_logs_path = f"*_bootstrap_*.tar/*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane/{host_ip}/journals/{journal_file}"
+    old_logs_path = f"*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/control-plane/{host_ip}/journals/{journal_file}"
```

Comment:
Consider having a const for the new and old logs_path prefix
e.g.
```
new_logs_path_prefix = f"*_bootstrap_*.tar/*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/"
old_logs_path_prefix = f"*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/"
 ```
 and use that everywhere,

Reply:
Done